game/templates/game/board.html

```
@@ -364,12 +364,12 @@
             </div>
 
             <div class="status-bar" id="statusBar">
-                <div id="replayControls" class="replay-controls hidden">
-                    <button id="firstReplayBtn">⏮</button>
-                    <button id="prevReplayBtn">◀</button>
-                    <button id="playReplayBtn">▶</button>
-                    <button id="nextReplayBtn">▶▶</button>
-                    <button id="lastReplayBtn">⏭</button>
+                <div id="replayControls" class="replay-controls hidden" aria-label="Game Replay Controls" role="group">
+                    <button id="firstReplayBtn" aria-label="Go to first move">⏮</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)